src/main/js/oauth.js

```diff
@@ -292,6 +292,15 @@ gromit.oAuthAuthenticate = function(/*String*/ url, /*function*/ callback, /*boo
     url += '?response_type=token';
     url += '&redirect_uri=' + gromit.getCurrentUrl() + 'oauth.html';
     url += '&client_id=' + gromit.ClientId;
+    
+    if (typeof gromit.AuthParams !== 'undefined') {
```

Comment:
This isn't a great way to check if something is defined in JavaScript.  You can just say `if (gromit.AuthParams)`

Reply:
Yeah, whatever. I'm not exactly a JS wizard.